samples/bluetooth/mesh/light/README.rst

```diff
@@ -65,6 +65,7 @@ The sample supports the following development kits:
 
       CONFIG_BT_CTLR_TX_BUFFER_SIZE=74
       CONFIG_BT_CTLR_DATA_LENGTH_MAX=74
+      CONFIG_BT_LL_SW_SPLIT=y
```

Comment:
I think it would be better to add these to a boards/nrf5384dk_netcore/prj.conf and then remove this note.

Reply:
We would like to default to the SoftDevice Controller for 53 applications as well. But not for Mesh

Reply:
Since this is a mesh sample then having that config in prj.conf should be fine, or do I misunderstand what you mean?

Reply:
Does this work for the sub-image build? If it does, I agree

Reply:
No, you would have to do something like how `bt_ll_softdevice_hci_rpmsg.conf` works.
See: `samples/CMakeLists.txt`